### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by armenak-baburyan**
Update due previous changes of docker-compose.yml.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,15 +14,13 @@ will get you up and running in no time!
...
-4. `docker-compose run --rm web upgrade` - Build the database.
+1. `docker-compose build` - Build and tag the Docker services
+2. `docker-compose run --rm web config generate-secret-key` - Generate a secret key.
+    Add it to `docker-compose.yml` in `x-defaults` as `SENTRY_SECRET_KEY`.
+3. `docker-compose run --rm web upgrade` - Build the database.
     Use the interactive prompts to create a user account.
-5. `docker-compose up -d` - Lift all services (detached/background mode).
-6. Access your instance at `localhost:9000`!
+4. `docker-compose up -d` - Lift all services (detached/background mode).
+5. Access your instance at `0.0.0.0:9000`!
```
Comment (mattrobenolt) [inline README.md]: Why this change? I disagree with this one. It doesn’t even make sense to use 0.0.0.0 from the client. I understand it works, but why not localhost anymore? 0.0.0.0 is unlikely to work with everything.

Comment (kanemra) [reply README.md]: To make clear that IP address may be external as well. May be use `IP_ADDRESS:9000` instead. localhost led me astray. Any way I can get `localhost` back.

Comment (revolunet) [reply README.md]: i suggest `http://127.0.0.1:9000` which is clickable :)

**Comment by lschricke** [comment]
This update looks good to me! 👍

**Comment by mattrobenolt** [comment]
Is any of this still relevant after https://github.com/getsentry/onpremise/pull/123 ?

**Comment by BYK** [comment]
Closing due to inactivity (lack of any responses to @mattrobenolt's latest comment). If anyone thinks we should look into this, please ping and I'll happily reopen.
